common/autoware_time_utils/src/time_utils/time_utils.cpp

```diff
@@ -17,6 +17,8 @@
 #include <limits>
 #include <stdexcept>
 
+// change
```

Comment:
Can you also make a change in a package from perception to see if ignore is working?

Reply:
Current `report.log`:

```log
Enabled checks:
    bugprone-dangling-handle

clang-apply-replacements version 14.0.0
clang-tidy-14 --use-color -export-fixes /tmp/tmpyffx98p7/tmpulc2pgnq.yaml -p=build/ /__w/autoware.universe/autoware.universe/common/autoware_time_utils/src/time_utils/time_utils.cpp
Writing fixes to /tmp/clang-tidy-result/fixes.yaml ...
```

Reply:
Tested in [here](https://github.com/autowarefoundation/autoware.universe/actions/runs/12065875880/job/33645777711?pr=9498)!
![Screenshot from 2024-11-28 18-27-24](https://github.com/user-attachments/assets/8ead66e4-8be9-4098-9069-8b740cf1d743)